exercises/perfectShuffle.js

```diff
@@ -8,7 +8,13 @@
  */
 
 function perfectShuffle(array) {
-  // This is your job. :)
+let half = array.length/2
+let perfectlyShufledArray=[];
+for (a = 0; a <half; a++){
+perfectlyShufledArray.push(array[a]);
```

Comment:
Great!

Press tab to indent code, though, this is _really_ hard for me to read.